src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type ErrorMessageProps = {
+  text?: string;
+  id?: string;
+  className?: string;
+  visuallyHiddenText?: string;
+  children?: ReactNode;
+};
+
+const ErrorMessage: FunctionComponent<ErrorMessageProps> = ({
```

Comment:
```suggestion
export const ErrorMessage: FunctionComponent<ErrorMessageProps> = ({
```

You'll need this for `export * from "./components/ErrorMessage";` to include it. That syntax doesn't include default exports.